src/tests/system/tests/test_ipa.py

```diff
@@ -122,3 +124,70 @@ def test_ipa__hostpublickeys_by_ip(client: Client, ipa: IPA, public_keys: list[s
     assert len(public_keys) == len(result.stdout_lines), "Did not get expected number of public keys!"
     for key in public_keys:
         assert f"{ip} {key}" in result.stdout_lines, "Did not get expected public keys!"
+
+
+@pytest.mark.ticket(bz=1926622)
+@pytest.mark.importance("critical")
```

Comment:
This is not a critical test and it's not a functional, please set the importance to low and add @pytest.mark.integration